src/GitHub.VisualStudio/Helpers/ActiveDocumentSnapshot.cs

```diff
@@ -19,8 +19,14 @@ public class ActiveDocumentSnapshot : IActiveDocumentSnapshot
         public ActiveDocumentSnapshot([Import(typeof(SVsServiceProvider))] IServiceProvider serviceProvider)
         {
             StartLine = EndLine = -1;
-            Name = Services.Dte2?.ActiveDocument?.ProjectItem.Name;
+            Name = Services.Dte2?.ActiveDocument?.FullName;
+            var projectItem = Services.Dte2?.ActiveDocument?.ProjectItem;
 
+            if ((String.Compare(Name, projectItem.FileNames[1], StringComparison.Ordinal) != 0) && (String.Compare(Name, projectItem.FileNames[1], StringComparison.OrdinalIgnoreCase) == 0))
```

Comment:
If the `FileName` and `Name` are the same in the first comparison then it's safe to assume we have the proper casing. But I guess that check is done in the second comparison as well. This could probably be condensed with a one line conditional operator for `Name` instead of possibly changing the value of `Name` in the `if` statement.